namedquery/src/main/java/software/amazon/athena/namedquery/UpdateHandler.java

```diff
@@ -1,28 +0,0 @@
-package software.amazon.athena.namedquery;
-
-import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
-import software.amazon.cloudformation.proxy.HandlerErrorCode;
-import software.amazon.cloudformation.proxy.Logger;
-import software.amazon.cloudformation.proxy.OperationStatus;
-import software.amazon.cloudformation.proxy.ProgressEvent;
-import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
-
-public class UpdateHandler extends BaseHandler<CallbackContext> {
-
-    @Override
-    public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
-        final AmazonWebServicesClientProxy proxy,
-        final ResourceHandlerRequest<ResourceModel> request,
-        final CallbackContext callbackContext,
-        final Logger logger) {
-
-        logger.log(String.format("%s [%s] calling dummy update handler",
-            ResourceModel.TYPE_NAME, request.getDesiredResourceState().getNamedQueryId()));
-
-        return ProgressEvent.<ResourceModel, CallbackContext>builder()
```

Comment:
do you not need dummy update handler anymore?

Reply:
A while back I believe the dummy update handler was required. When I was reading through the documentation I noticed that update handler is optional now, and if it doesn't exist then any change automatically triggers replacement (which is the correct behavior for NamedQuery as they are not mutable within Athena). So in this case we shouldn't need the update handler